man/man5/zfs-module-parameters.5

```diff
@@ -1955,12 +1955,12 @@ Default value: \fB52,428,800\fR.
 \fBzfs_per_txg_dirty_frees_percent \fR (ulong)
 .ad
 .RS 12n
-Tunable to control percentage of dirtied blocks from frees in one TXG.
-After this threshold is crossed, additional dirty blocks from frees
-wait until the next TXG.
+Tunable to control percentage of dirtied indirect blocks from frees allowed
+into one TXG. After this threshold is crossed, additional dirty blocks
+from frees will wait until the next TXG.
```

Comment:
nit: `dirty blocks from frees` -> `frees`?  Since the free is the user-understandable operation that's waiting.